packages/template-retail-react-app/app/components/store-locator-modal/stores-list.jsx

```diff
@@ -5,7 +5,7 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import React from 'react'
```

Comment:
You're missing a `CHANGELOG.md` update associated with this PR

Reply:
Added! @bfeister question on the release process. Currently I have the base branch set to a feature branch `feature-bopis`. Does this make sense or should I just merge the change into `develop`?

This change does works on it's own, although it doesn't really do anything other than save the store info in local storage

Reply:
I think it's fine to merge into `develop` since it works on it's own. It'll help to avoid a "big bang" merge and make that diff smaller